build/azure-pipelines/linux/sql-product-build-linux.yml

```diff
@@ -129,16 +129,19 @@ steps:
   - script: |
       set -e
       yarn gulp vscode-linux-x64-build-deb
+      du -ah ../ | sort -n -r
```

Comment:
So with the -h option sorting by numerical value isn't really helpful here since 1024K will be sorted before 1G